v2/cmd/asoctl/go.mod

```diff
@@ -25,12 +25,12 @@ require (
 	k8s.io/apiextensions-apiserver v0.32.1
 	k8s.io/apimachinery v0.32.1
 	k8s.io/client-go v0.32.1
-	sigs.k8s.io/controller-runtime v0.19.5
+	sigs.k8s.io/controller-runtime v0.20.2
```

Comment:
I think this version change is going to cause CI issues, see: https://github.com/Azure/azure-service-operator/issues/4381

Reply:
Can you bump it back down to 19.5?